src/write_redis.c

```diff
@@ -45,6 +49,7 @@ struct wr_node_s
   int port;
   struct timeval timeout;
   char *prefix;
+  char *database;
```

Comment:
If I read the docs correctly, this is a numeric id. Would it make sense to use an `int` here (and `cf_util_get_int()` below)?